packages/cli/src/commands/init/utils.ts

```diff
@@ -56,249 +55,68 @@ export const getModelIdentifier = (llmProvider: LLMProvider) => {
-      - Always ask for a location if none is provided
-      - If the location name isn't in English, please translate it
-      - If giving a location with multiple parts (e.g. "New York, NY"), use the most relevant part (e.g. "New York")
-      - Include relevant details like humidity, wind conditions, and precipitation
-      - Keep responses concise but informative
-      - If the user asks for activities and provides the weather forecast, suggest activities based on the weather forecast.
-      - If the user asks for activities, respond in the format they request.
-
-      ${addExampleTool ? 'Use the weatherTool to fetch current weather data.' : ''}
-`;
-  const content = `
-import { Agent } from '@mastra/core/agent';
-import { Memory } from '@mastra/memory';
-import { LibSQLStore } from '@mastra/libsql';
-${addExampleTool ? `import { weatherTool } from '../tools/weather-tool';` : ''}
-
-export const weatherAgent = new Agent({
-  name: 'Weather Agent',
-  instructions: \`${instructions}\`,
-  model: ${modelString},
-  ${addExampleTool ? 'tools: { weatherTool },' : ''}
-  memory: new Memory({
-    storage: new LibSQLStore({
-      url: "file:../mastra.db", // path is relative to the .mastra/output directory
-    })
-  })
-});
-    `;
-  const formattedContent = await prettier.format(content, {
-    parser: 'typescript',
-    singleQuote: true,
-  });
-
-  await fs.writeFile(destPath, '');
-  await fs.writeFile(destPath, formattedContent);
-}
-
-export async function writeWorkflowSample(destPath: string) {
-  const content = `import { createStep, createWorkflow } from '@mastra/core/workflows';
-import { z } from 'zod';
-
-const forecastSchema = z.object({
-  date: z.string(),
-  maxTemp: z.number(),
-  minTemp: z.number(),
-  precipitationChance: z.number(),
-  condition: z.string(),
-  location: z.string(),
-})
-
-function getWeatherCondition(code: number): string {
-  const conditions: Record<number, string> = {
-    0: 'Clear sky',
-    1: 'Mainly clear',
-    2: 'Partly cloudy',
-    3: 'Overcast',
-    45: 'Foggy',
-    48: 'Depositing rime fog',
-    51: 'Light drizzle',
-    53: 'Moderate drizzle',
-    55: 'Dense drizzle',
-    61: 'Slight rain',
-    63: 'Moderate rain',
-    65: 'Heavy rain',
-    71: 'Slight snow fall',
-    73: 'Moderate snow fall',
-    75: 'Heavy snow fall',
-    95: 'Thunderstorm',
-  }
-  return conditions[code] || 'Unknown'
-}
-
-const fetchWeather = createStep({
-  id: 'fetch-weather',
-  description: 'Fetches weather forecast for a given city',
-  inputSchema: z.object({
-    city: z.string().describe('The city to get the weather for'),
-  }),
-  outputSchema: forecastSchema,
-  execute: async ({ inputData }) => {
-    if (!inputData) {
-      throw new Error('Input data not found');
-    }
+// Resolve a template file from templates/weather-agent/src/mastra
+const readWeatherTemplateFile = async (subpath: string): Promise<string> => {
```

Comment:
Can't we adjust the `copyStarterFile` method on the FileService class and use that? If the template uses the model router syntax then you also don't need to adjust the string in this file here